pkg/ccl/backupccl/restore_online_test.go

```diff
@@ -195,16 +232,25 @@ func TestOnlineRestoreWaitForDownload(t *testing.T) {
 	externalStorage := "nodelocal://1/backup"
 
 	sqlDB.Exec(t, fmt.Sprintf("BACKUP INTO '%s'", externalStorage))
+	sqlDB.Exec(t, `SET CLUSTER SETTING jobs.debug.pausepoints = 'restore.before_do_download_files'`)
 
 	sqlDB.Exec(t, fmt.Sprintf("RESTORE DATABASE data FROM LATEST IN '%s' WITH EXPERIMENTAL DEFERRED COPY, new_db_name=data2", externalStorage))
 
 	// Wait for the download job to complete.
 	var downloadJobID jobspb.JobID
 	sqlDB.QueryRow(t, `SELECT job_id FROM [SHOW JOBS] WHERE description LIKE '%Background Data Download%'`).Scan(&downloadJobID)
+	jobutils.WaitForJobToPause(t, sqlDB, downloadJobID)
```

Comment:
Pausing, waiting, resuming and doing more waiting here seems like it just slows this test down, and our tests are already painfully slow.

Reply:
This takes 2s, like every other test in the package. I also think its important to verify that this utility query returns nonzero results.

Reply:
Lemme randomly pause one of the existing tests, if that makes you feel better.

Reply:
I donno: 2s here, 2s there, and then under race or stress that becomes 40s and we end up with `backupccl` being a monster package with slow flaky tests.

I think verifying something like "does this one span stats rpc work" by pausing and resuming a whole job is exactly why we have the slowest test packages in the codebase?

Reply:
ok fine, i'll test the rpc once i get the roachtests up and running again.